webrtc/RTCIceTransport.html

```diff
@@ -114,81 +65,128 @@
     assert_true(candidatePair.remote instanceof RTCIceCandidate,
       'Expect candidatePair.remote to be instance of RTCIceCandidate');
 
-    validateCandidateParameter(iceTransport.getLocalParameters());
-    validateCandidateParameter(iceTransport.getRemoteParameters());
+    validateIceParameter(iceTransport.getLocalParameters());
+    validateIceParameter(iceTransport.getRemoteParameters());
+  }
+
+  function validateUnconnectedIceTransport(iceTransport) {
+    assert_array_equals(iceTransport.getRemoteCandidates(), [],
+      'Expect iceTransport to not have any remote candidate');
+
+    assert_equals(iceTransport.getSelectedCandidatePair(), null,
+      'Expect selectedCandidatePair to be null');
   }
 
-  promise_test(t => {
+  promise_test(async t => {
+    const pc1 = new RTCPeerConnection();
+    t.add_cleanup(() => pc1.close());
+
+    const pc2 = new RTCPeerConnection();
+    t.add_cleanup(() => pc2.close());
+
+    pc1.createDataChannel('');
+    exchangeIceCandidates(pc1, pc2);
+
+    await doSignalingHandshake(pc1, pc2);
+
+    const iceTransports = [
+      pc1.sctp.transport,
+      pc2.sctp.transport
+    ]
+    .filter(dtlsTransport => dtlsTransport) // filter null/undefined
+    .map(dtlsTransport => dtlsTransport.iceTransport);
+
+    assert_greater_than_equal(iceTransports.length, 2, 'expect to find at least 2 RTCIceTransports');
```

Comment:
Nit: Wouldn't it be fewer lines to just assert each of the two `pcN.sctp.transport.iceTransport`s?